libs/properties/src/main/java/org/padaiyal/utilities/PropertyUtility.java

```diff
@@ -54,7 +55,7 @@ public static synchronized String getProperty(String propertyKey) {
                 })
             .filter(Objects::nonNull)
             .findFirst()
-            .orElseThrow();
+            .orElseThrow(NoSuchElementException::new);
```

Comment:
What was the motivation for this change in the properties package?

Reply:
unit test was failing without this.
By default, it throws InvalidArgumentException, but the junit was expecting NoSuchElementException.

Reply:
Which test was failing?
If it's in filesystem, then it should be changed to expect an InvalidArgumentException.
If it's in properties, then there should be a pre-existing failure which wasn't the case.

Reply:
it's a junit in properties. Not in filesystem.
Let me share the log from the github actions, the next time i see it.

Reply:
Reverted this. Not an issue.